CONTRIBUTING.md

```diff
@@ -18,10 +18,38 @@ git push -f
 
 It's okay to force push over your own PR branch. In fact, that's what we want so that the commits are clean.
 
-
 ## How to squash a PR
-See [this how to](https://twitter.com/stephenfluin/status/1009904095073718275).
+To squash multiple commits, use the following steps:
+
+### 1. Rebase multiple commits into one
+
+When squashing commits, you should always rebase from the main branch of the repo: `git rebase -i origin/main`.
+
+After rebasing, you can squash the commits by changing `pick` into `squash` for all of your commits except the recent one:
+```shell
+pick 686e386 add doc structure
+squash ee2eed7 add main content
+squash 3eccd39 fix typos
+
+# Rebase 287bc79..3eccd39 onto 287bc79 (3 commands)
+# …
+```
+
+### 2. Fix your commit message
+
+Next, comment or remove all of your commit messages and leave the one that describes your PR: `add doc X`, `fix page Y`, `improve docs for Z`. For example: `add contract addresses for X blockchain`. To save the changes, use `wq`.
```

Comment:
`wq` will depend if their system is setup to use VI/VIM or EMACS or a GUI editor. Let's not try to teach VI here. :)